extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -183,6 +185,7 @@ void waitForVertxPool(List<VertxPoolBuildItem> vertxPool,
 
     @BuildStep
     @Record(RUNTIME_INIT)
+    @Produce(SyntheticBeansRuntimeInitBuildItem.class)
```

Comment:
@mkouba grpc is initialized after this build item, and it sounds like something that should be safe ;) Do you know if there is build item that could be to express this order?

Reply:
I didn't want to have explicit dependency between HR and gRPC.

Reply:
Not sure if it's useful in this case but for some other purposes we are producing a `ServiceStartBuildItem` in Hibernate ORM.

Reply:
I don't think I can use it, the `GrpcServerProcessor` step that I want to make sure runs after this step is producing `ServiceStartBuildItem` itself.

Reply:
@michalszynkiewicz You can't produce a `SyntheticBeansRuntimeInitBuildItem`. It can be only used to make sure a build step is executed AFTER the `RUNTIME_INIT` synthetic beans are initialized.

`ServiceStartBuildItem`, on the other hand, is used to make sure a build step is exectued before the `StartupEvent` is fired.